Sprint-1/destructuring/exercise-1/exercise.js

```diff
@@ -6,10 +6,24 @@ const personOne = {
 
 // Update the parameter to this function to make it work.
 // Don't change anything else.
-function introduceYourself(___________________________) {
+function introduceYourself({name,age,favouriteFood}) {
   console.log(
     `Hello, my name is ${name}. I am ${age} years old and my favourite food is ${favouriteFood}.`
   );
 }
 
 introduceYourself(personOne);
+
+/*
+or: 
+
+function introduceYourself({name,age,favouriteFood}) {
+  const {name,age,favouriteFood} = personOne;
```

Comment:
This is alternative function. The function gets destructuring values from personOne object as parameters on line 20 and then again does the same on line 21. Why do you think it is needed?